webgpu.h

```diff
@@ -529,9 +530,9 @@ typedef struct WGPUBindGroupEntry {
 } WGPUBindGroupEntry;
 
 typedef struct WGPUBlendComponent {
+    WGPUBlendOperation operation;
     WGPUBlendFactor srcFactor;
     WGPUBlendFactor dstFactor;
-    WGPUBlendOperation operation;
 } WGPUBlendComponent;
```

Comment:
nit: spec actually has the original order: srcFactor, dstFactor, operation.

I'm going to open a PR with a few other things so I'll include it.

Reply:
oops